packages/babel-types/src/index.js

```diff
@@ -399,6 +405,10 @@ export function inherits(child, parent) {
 
   t.inheritsComments(child, parent);
 
+  const pathCache = traverse.cache.path;
+  if (pathCache.has(parent)) {
+    pathCache.set(child, pathCache.get(parent));
+  }
```

Comment:
Interesting, do you happen to know an example where this affects things? I could see it being a performance win, but I'm trying to think if there's other cases.